packages/patternfly-4/react-core/src/components/Modal/Modal.d.ts

```diff
@@ -9,6 +9,7 @@ export interface ModalProps extends HTMLProps<HTMLDivElement> {
   isOpen?: boolean;
   onClose?: Function;
   title: string;
+  srHide: Element;
```

Comment:
Can we please make srHide optional? IMO, patternfly should make it easy to implement a11y, but not force it.

Not sure why this worked during my test? That is, I didn't provide an srHide property.